x-pack/plugins/security/server/prompt_page.tsx

```diff
@@ -65,7 +65,7 @@ export function PromptPage({
           <EuiPageBody>
             <EuiPageContent verticalPosition="center" horizontalPosition="center">
               <EuiEmptyPrompt
-                iconType="alert"
```

Comment:
<img width="803" alt="Screenshot 2023-03-08 at 10 14 05 AM" src="https://user-images.githubusercontent.com/21210601/223752429-5d49ea6a-9583-499b-a668-6cb68c56a6b4.png">

The icon does not appear properly on this page.

Reply:
Should be fixed now in 1351cba - thanks a million for catching this @kc13greiner!